docs/groovy/reference/community-questions/shifts.md

```diff
@@ -0,0 +1,129 @@
+---
+title: How do row keys and positional indices behave during merge table operations?
+sidebar_label: How do row keys and positional indices behave during merge table operations?
+---
+
+_When merging tables in Deephaven, I understand that "shifts" can happen when one table grows beyond its allocated row key slots. How do the row key (k) and positional index (i) attributes change during these shifts, and what impact does this have on downstream operations?_
```

Comment:
This is a pretty low-level concept that I doubt even 1% of users know about outside of our devs. For example, I don't really know what this means. This feels like it needs to define some of the phrases used, otherwise a reader who stumbles on this will be lost. Where did this CQ come from?

This comment isn't to say we shouldn't have this CQ page, but is this the right place for this info? This seems like material worthy of a conceptual guide or a section in a conceptual guide somewhere.